howso/client/tests/test_client.py

```diff
@@ -1109,26 +1105,16 @@ def test_get_num_training_cases(self, trainee):
         number_cases = self.client.get_num_training_cases(trainee.id)
         assert isinstance(number_cases, int)
 
-    def test_react_group_exception(self, trainee):
-        """
-        Test that react_group raises when expected.
-
-        Test the expected exception is raised when react_group
-        gets parameters trainee_id and trainee_to_compare passed in as None.
-        """
-        with pytest.raises(ValueError) as exc:
-            self.client.react_group(trainee.id)
-        assert str(exc.value) == ("Either `new_cases` or `trainees_to_compare` "
-                                  "must be provided.")
-
     def test_react_group_trainee_compare_verbose(self, trainee, capsys):
```

Comment:
Not sure we need this test anymore now that trainee_compare is removed. We already have test above for new_cases.